### PR TITLE
Patch 2

### PR DESCRIPTION
**Pull request by Grammaresque**
#### Author checklist (Completed by original Author)
- [ ] Good fit for the Rocky Linux project? Title and Author Metatags inserted ?
- [ ] If applicable, steps and instructions have been tested to work
- [ ] Initial self-review to fix basic typos and grammar completed

#### Rocky Documentation checklist (Completed by Rocky team) 
- [ ] 1st Pass (Document is good fit for project and author checklist completed)
- [ ] 2nd Pass (Technical Review - check for technical correctness) 
- [ ] 3rd Pass (Detailed Editorial Review and Peer Review)
- [x] Final approval (Final Review)

### CODE REVIEW

File: docs/books/admin_guide/01-presentation.md [hunk 1]
```diff
@@ -53,130 +53,130 @@ Among this management of resources, the operating system has to:
...
+* **2021**: Birth of Rocky Linux, based on Red Hat distribution.
 
 ### Market share
 
 <!--
 TODO: graphics with market share for servers and pc.
 -->
 
-Linux is still not well known by the general public, even though they use it regularly. Indeed, Linux is hidden in **smartphones**, **televisions**, **internet boxes**, etc. Almost **70% of the web pages** served in the world are served by a Linux or UNIX server!
+Despite its prevalence, Linux remains relatively unknown by the general public, . Linux is hidden in **smartphones**, **televisions**, **internet boxes**, etc. Almost **70% of the websites** in the world are hosted on a Linux or UNIX server!
```
Comment (SergeCroise) [inline docs/books/admin_guide/01-presentation.md:95]:
```suggestion
Despite its prevalence, Linux remains relatively unknown by the general public. Linux is hidden in **smartphones**, **televisions**, **internet boxes**, etc. Almost **70% of the websites** in the world are hosted on a Linux or UNIX server!
```

File: docs/books/admin_guide/01-presentation.md [hunk 2]
```diff
@@ -53,130 +53,130 @@ Among this management of resources, the operating system has to:
...
 
-The **Gnome 3** desktop environment no longer uses the concept of Desktop but that of Gnome Shell (not to be confused with the command line shell). It serves as a desktop, a dashboard, a notification area and a window selector. The Gnome desktop environment is based on the GTK+ component library.
+The **GNOME 3** desktop environment no longer uses the concept of desktop but that of GNOME Shell (not to be confused with the command line shell). It serves as a desktop, a dashboard, a notification area and a window selector. The GNOME desktop environment is based on the **GTK+** component library.
 
 ![KDE Desktop](images/01-presentation-kde.png)
 
 The **KDE** desktop environment is based on the **Qt** component library.
 
-It is traditionally more recommended for users coming from a Windows world.
+It is traditionally recommended for users familiar with a Windows environmnent.
```
Comment (SergeCroise) [inline docs/books/admin_guide/01-presentation.md:179]:
```suggestion
It is traditionally recommended for users familiar with a Windows environment.
```

File: docs/books/admin_guide/01-presentation.md [hunk 3]
```diff
@@ -188,22 +188,22 @@ In the **GNU/Linux** world, the Free Software movement provides mostly free dist
 
 **Free** does not mean free!
 
-**Open source**: the source codes are available, so it is possible to consult and modify them under certain conditions.
+**Open Source**: the source codes are available, so it is possible to consult and modify them under certain conditions.
```
Comment (SergeCroise) [inline docs/books/admin_guide/01-presentation.md:191]:
```suggestion
**Open Source**: the source code is available, so it is possible to consult and modify them under certain conditions.
```

File: docs/books/admin_guide/01-presentation.md [hunk 4]
```diff
@@ -188,22 +188,22 @@ In the **GNU/Linux** world, the Free Software movement provides mostly free dist
...
 
 A free software is necessarily Open Source, but the opposite is not true since an Open Source software is separated from the freedom proposed by the GPL license.
 
-#### GPL License (General Public License)
+#### GPL (General Public License)
 
-The **GPL License** guarantees the author of a software its intellectual property, but allows modification, redistribution or resale of software by third parties, provided that the source codes are provided with the software. The GPL is the license that came out of the **GNU** (GNU is Not UNIX) project, which was instrumental in creating Linux.
+The **GPL** guarantees the author of a software its intellectual property, but allows modification, redistribution or resale of software by third parties, provided that the source codes are included with the software. The GPL is the license that came out of the **GNU** (GNU is Not UNIX) project, which was instrumental in creating Linux.
```
Comment (SergeCroise) [inline docs/books/admin_guide/01-presentation.md:197]:
```suggestion
The **GPL** guarantees the author of a software its intellectual property, but allows modification, redistribution or resale of software by third parties, provided that the source code is included with the software. The GPL is the license that came out of the **GNU** (GNU is Not UNIX) project, which was instrumental in creating Linux.
```

**Review by wsoyinka — APPROVED** [review]

File: docs/books/admin_guide/01-presentation.md [hunk 5]
```diff
@@ -224,20 +224,19 @@ The **shell**, known as _command interface_, allows users to send commands to th
...
-* Command execution (checks the command given and executes it);
-* Input/Output redirection (returns data to a file instead of writing it on the screen);
-* Connection process (manages the user's connection);
-* Interpreted programming language (allowing the creation of scripts);
-* Environment variables (access to information specific to the system during operation).
+* Command execution (checks the command given and executes it)
+* Input/Output redirection (returns data to a file instead of writing it on the screen)
+* Connection process (manages the user's connection)
+* Interpreted programming language (allowing the creation of scripts)
+* Environmental variables (access to information specific to the system during operation)
```
Comment (SergeCroise) [inline docs/books/admin_guide/01-presentation.md:239]:
```suggestion
* Environment variables (access to information specific to the system during operation)
```

**Comment by wsoyinka** [comment]
Thank you @Grammaresque  :-)

**Pull request merged**